src/zenml/zen_stores/schemas/component_schemas.py

```diff
@@ -85,6 +86,13 @@ class StackComponentSchema(NamedSchema, table=True):
     run_metadata: List["RunMetadataSchema"] = Relationship(
         back_populates="stack_component",
     )
+    flavor_schema: Optional["FlavorSchema"] = Relationship(
```

Comment:
Can we not remove the `flavor: str` field now that we have the explicit reference here, and then call this attribute `flavor` to be consistent? Or is that out of scope for this ticket?

Reply:
Out of scope for this one.